packages/addon-dev/sample-rollup.config.js

```diff
@@ -1,15 +1,28 @@
-import babel from '@rollup/plugin-babel';
+import { babel } from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
 import { Addon } from '@embroider/addon-dev/rollup';
 
+import packageJson from './package.json';
+
 const addon = new Addon({
   srcDir: 'src',
   destDir: 'dist',
 });
 
+const extensions = ['.js', '.ts'];
```

Comment:
If we remove `nodeResolve`, I think we ought to be able move this to inline with `babel` with a nice little comment?